### DISCUSSION TITLE
How to calculate the stress after model excavation?

### DISCUSSION THREAD

**Discussion by zuoyu158** [gempy-project/gempy#654] [Q&A]
How to show the stress cloud map after excavation? Can you share the code if you have it? Thank you.

**Comment by AlexanderJuestel** [comment]
Hello,

GemPy is not made for geomechanical modeling. What kind of stress do you want to calculate?

Cheers Alex

**Reply by zuoyu158** [reply]
Hi，
Thank you for your reply. We have learned that gempy cannot calculate geological forces. We calculate geological forces by ourselves. Can we display the calculated values in different colors in different layers?
Thank you

**Reply by AlexanderJuestel** [reply]
Hello @zuoyu158,

you could export the meshes of your model and set new scalar values for the meshes. So, currently, each mesh has a certain color but you could colorize each mesh according to your calculated stress values by setting a scalar value. 

Cheers Alex

**Reply by zuoyu158** [reply]
Thank you for your reply. Can the established model be exported as a .f3grid file.

**Reply by AlexanderJuestel** [reply]
Hey,

I am not familiar with this file format and I am quite sure that GemPy does not support it.

**Reply by zuoyu158** [reply]
Hi,
Thank you very much for your reply. How to  export the meshes of your model and set new scalar values for the meshes? If there are relevant examples or codes, can you share them?
Thank you.
